src/js/themes/base.d.ts

```diff
@@ -812,8 +836,8 @@ export interface ThemeType {
     };
     options?: {
       container?: {
-        align?: string;
-        pad?: string;
+        align?: BoxProps['align'];
+        pad?: BoxProps['pad'];
```

Comment:
my bad, let's roll it back to how it was.

Reply:
Thanks, all set :)